dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/QuayImageRegistry.java

```diff
@@ -133,11 +158,45 @@ public QuayImageRegistry(final Token quayToken) {
         return tags;
     }
 
-    private Tag convertQuayTagToTag(QuayTag quayTag, Tool tool) throws InvocationTargetException, IllegalAccessException {
+    // For each manifest in the list:
```

Comment:
Real minor, but if you're commenting a method, you should generally use JavaDoc commenting.